internal/service/dataexchange/event_action_test.go

```diff
@@ -0,0 +1,580 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package dataexchange_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/service/dataexchange"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry"
+	"github.com/hashicorp/terraform-plugin-testing/compare"
+	sdkacctest "github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/statecheck"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/hashicorp/terraform-plugin-testing/tfjsonpath"
+	"github.com/hashicorp/terraform-provider-aws/internal/acctest"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+	"github.com/hashicorp/terraform-provider-aws/internal/create"
+	"github.com/hashicorp/terraform-provider-aws/internal/errs"
+	tfdataexchange "github.com/hashicorp/terraform-provider-aws/internal/service/dataexchange"
+	"github.com/hashicorp/terraform-provider-aws/names"
+)
+
+const testAccDataSetIDEnvVar = "TF_AWS_DATAEXCHANGE_DATA_SET_ID"
```

Comment:
Can we add this to the contributor docs?
https://hashicorp.github.io/terraform-provider-aws/acc-test-environment-variables/